modules/images-other-jenkins-config-kubernetes.adoc

```diff
@@ -6,10 +6,19 @@
 [id="images-other-jenkins-config-kubernetes_{context}"]
 = Configuring the Jenkins Kubernetes plug-in
 
-The {product-title} Jenkins image includes the pre-installed link:https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin[Kubernetes plug-in] that allows Jenkins agents to be dynamically provisioned on multiple container hosts using Kubernetes and {product-title}.
+The OpenShift Jenkins image includes the pre-installed link:https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin[Kubernetes plug-in] so that Jenkins agents can be dynamically provisioned on multiple container hosts using Kubernetes and {product-title}.
 
 To use the Kubernetes plug-in, {product-title} provides images that are suitable for use as Jenkins agents, including the Base, Maven, and Node.js images.
 
+[IMPORTANT]
+====
+{product-title} 4.11 moves the "OpenShift Jenkins" and "OpenShift Agent Base" images to the `ocp-tools-4` repository at `registry.redhat.io` so that Red Hat can produce and update the images outside the {product-title} lifecycle. Previously, these images were in the {product-title} install payload and the `openshift4` repository at `registry.redhat.io`.
```

Comment:
```suggestion
{product-title} 4.11 moves the OpenShift Jenkins and OpenShift Agent Base images to the `ocp-tools-4` repository at `registry.redhat.io` so that Red Hat can produce and update the images outside the {product-title} lifecycle. Previously, these images were in the {product-title} install payload and the `openshift4` repository at `registry.redhat.io`.
```

The use of the double-quotation marks here doesn't fall into any of the use cases cited in the ISG:

```

- Quotations.
- Words that are used in a special sense, but only if no other word clearly expresses the meaning. Do not overuse quotation marks for this purpose. Using a word in a special sense can cause problems for translation.
- Citations that have sentence-style capitalization and do not have any other highlighting.
- Hardware label names.
- Interface elements that have sentence-style capitalization and that need to be distinguished from surrounding text.

```